k-distribution/tests/regression-new/macro_vars-productions/macro-vars.k

```diff
@@ -12,4 +12,7 @@ module MACRO-VARS
   syntax CPPType ::= underlyingType(CPPType) [function]
   syntax CPPType ::= "cppEnumType" [macro]
   rule cppEnumType => bar(_, scopedEnum() #Or unscopedEnum() ) [unboundVariables(_)]
+  syntax Int ::= A ( Int ) [macro]
+  rule A ( N ) => N +Int 1
+  rule A ( N ) => N [simplification]
```

Comment:
This is a bit weird of a definition, you're basically saying that `N ==Int N +Int 1`. Not sure it matters because this is a test, but it looks weird to me.

Reply:
I agree. it's a really weird definition but this is also one of the simplest cases. There's also an error that occurs if I change N +Int 1 to N but I'll file a separate bug for that